test/user.test.js

```diff
@@ -2193,6 +2193,72 @@ describe('User', function() {
     });
   });
 
+  describe('Verification after updating email', function() {
+    var nonVerifiedUser;
```

Comment:
I find the variable name confusing. You are creating the user with `emailVerified` set to `true` and then store it in a variable called `nonVerifiedUser`.

Can you use a different name please? As far as I am concerned, `user`, `userInstance` or `testUser` are good enough. Feel free to come up with your own name though!